interface/modules/custom_modules/oe-module-weno/sql/table.sql

```diff
@@ -0,0 +1,89 @@
+  `NPI` varchar(10) DEFAULT NULL,
+  `NPI_safe` varchar(10) DEFAULT NULL,
+  `Business_Name` varchar(255) DEFAULT NULL,
+  `Address_Line_1` varchar(255) DEFAULT NULL,
+  `Address_Line_2` varchar(255) DEFAULT NULL,
+  `City` varchar(20) DEFAULT NULL,
+  `State` varchar(20) DEFAULT NULL,
+  `ZipCode` varchar(5) DEFAULT NULL,
+  `ZipCode_safe` varchar(10) DEFAULT NULL,
+  `Country_Code` varchar(255) DEFAULT NULL,
+  `International` tinyint(1) DEFAULT NULL,
+  `Latitude` varchar(255) DEFAULT NULL,
+  `Longitude` varchar(255) DEFAULT NULL,
+  `Pharmacy_Phone` varchar(255) DEFAULT NULL,
+  `Pharmacy_Phone_safe` varchar(255) DEFAULT NULL,
+  `Pharmacy_Fax` varchar(255) DEFAULT NULL,
+  `Types` varchar(255) DEFAULT NULL,
+  `Script_Msg_Accepted` varchar(255) DEFAULT NULL,
+  `Specialized_Msg_Accepted` varchar(255) DEFAULT NULL,
+  `Connectivity_Status` varchar(255) DEFAULT NULL,
+  `Accept_TSO` varchar(255) DEFAULT NULL,
+  `DEA_Audit_Exp` varchar(255) DEFAULT NULL,
+  `Test_Pharmacy` varchar(5) DEFAULT NULL,
+  `State_Wide_Mail_Order` varchar(6) NOT NULL,
+  `Created` datetime DEFAULT NULL,
+  `Modified` datetime DEFAULT NULL,
+  `Deleted` datetime DEFAULT NULL,
+  `24HR` varchar(3) DEFAULT NULL,
+  `on_weno` tinytext NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `id` (`id`),
+  UNIQUE KEY `ncpdp` (`NCPDP`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+#EndIf
+
+#IfNotTable weno_assigned_pharmacy
+CREATE TABLE IF NOT EXISTS `weno_assigned_pharmacy` (
+    `id` INT(10) NOT NULL AUTO_INCREMENT,
+    `pid` BIGINT(20) NOT NULL,
+    `primary_ncpdp` VARCHAR(8) NOT NULL,
+    `alternate_ncpdp` VARCHAR(8) NOT NULL,
+    PRIMARY KEY(id)
+) ENGINE=InnoDB;
+#EndIf
+
+#IfNotTable weno_download_log
+CREATE TABLE IF NOT EXISTS `weno_download_log` (
+    `id` BIGINT(20) NOT NULL AUTO_INCREMENT,
+    `value` VARCHAR(12) NOT NULL,
+    `status` VARCHAR(10) NOT NULL,
+    `created_at` DATE NOT NULL DEFAULT (CURRENT_DATE),
+    PRIMARY KEY (`id`),
+    KEY `value` (`value`)
+) ENGINE=InnoDB;
+#EndIf
+
+#IfNotRow background_services name WenoExchangePharmacies
+INSERT INTO `background_services` (`name`, `title`, `active`, `running`, `next_run`, `execute_interval`, `function`, `require_once`, `sort_order`) 
+VALUES ('WenoExchangePharmacies', 'Weno Exchange Pharmacy', '0', '0', current_timestamp(), '1440', 'downloadWenoPharmacy', '/weno_log_sync.php', '100');
+#EndIf
+
+#IfNotRow background_services name WenoExchange
+INSERT INTO `background_services` (`name`, `title`, `active`, `running`, `next_run`, `execute_interval`, `function`, `require_once`, `sort_order`) 
+VALUES ('WenoExchange', 'Weno Prescription Log', '0', '0', current_timestamp(), '30', 'downloadWenoPrescriptionLog', '/weno_log_sync.php', '100');
+#EndIf
+
+#IfRow globals name weno_provider_password
+ALTER TABLE globals RENAME COLUMN weno_provider_password TO weno_admin_password;
+#EndIf
+
+IfRow globals name weno_provider_username
+ALTER TABLE globals RENAME COLUMN weno_provider_username TO weno_admin_username;
```

Comment:
also, I am not seeing a global `weno_provider_username` in the current codebase or globals. Is there a reason why need to do this?

Reply:
`weno_provider_username` can be found in globals.inc.php. Individuals who have toggled weno settings in the globals section would have that in the database. Also, a patch was released for some clients last year which made use of the `weno_provider_username`.